elide-integration-tests/src/test/resources/EncodedErrorResponsesIT/InvalidAttributeException.req.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
This was the old way of doing tests in Elide.  There is a JSON-API DSL that is much easier to read where the bodies remain inside the tests (rather than buried in some files).